src/drivers/intel/cavs/ipc.c

```diff
@@ -167,7 +167,7 @@ void ipc_platform_do_cmd(struct ipc *ipc)
 	/* unmask Busy interrupt */
 	ipc_write(IPC_DIPCCTL, ipc_read(IPC_DIPCCTL) | IPC_DIPCCTL_IPCTBIE);
 
-#if CAVS_VERSION == CAVS_VERSION_2_0
+#if CAVS_VERSION >= CAVS_VERSION_2_0
```

Comment:
nit-pick: you may want to list 2.0 and 2.5, not sure how we are going to deal with future platforms.

Reply:
Usually most of configs are made in backward compatible way, if they are not then you just go and change if's like that. It's already like that in other parts of FW - at least it was like that for >=1.8, just 2.0 had specific check here, I wanted to make it consistent.